src/execution/gaugeExecution.ts

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import { Position } from 'vscode';
+import { window, Position, workspace, Uri, WorkspaceFolder } from 'vscode';
```

Comment:
WorkspaceFolder seems to be unused. 
Also, we are calling vscode.window in some places. Can we just consolidate them to use window now?